README.md

```diff
@@ -201,6 +201,11 @@ Below is a list of configuration options related to search, used in the [base co
       verticalKey: 'verticalKey',
       // Optional, the number of results to display per page, defaults to 20. Maximum is 50.
       limit: '20',
+      // Optional, an object containing the number of results to display per vertical key. Minimum is 1. Maximum is 50. No default provided.
+      universalLimit: {
```

Comment:
Is there an implicit limit in the sense that if no object is provided here, they will get max 50 results per vertical?

Reply:
I think the default is 10 because when I perform a universal query, I'm only seeing 10 results per vertical